.bit/responses/5.6-Week 5 Step 6.md

```diff
@@ -0,0 +1,32 @@
+---
+files: n/a
+stepType: checks
+scripts: n/a
+week: 5
+step: 6
+name: Week 5 Step 6
+---
+Week 5 Step 6 ◯◯◯◯◯⬤ | 🕐 Estimated completion: 25 minutes
+
+## Documentation
+
+### ✅  Task:
+
+- [ ] Document your first component in blog.md
+- [ ] Commit your changes in the `week5` branch
+
+Once you have successfully completed developing your first component, you can start documenting your project blog. Remember that the purpose of this blog is to help someone else create your project from scratch, so as you start writing, you will need to break down your code as much as possible and explain it so that someone else can learn from your example.
+
+### Quick tips
```

Comment:
Once again, I think solid examples would be super helpful (you could pull some from blogs that have already been written, or attach screenshots etc.)